Makefile

```diff
@@ -21,7 +21,7 @@ BUILDDIR ?= $(CURDIR)/build
 TEST_DOCKER_REPO=cosmos/contrib-gaiatest
 
 GO_SYSTEM_VERSION = $(shell go version | cut -c 14- | cut -d' ' -f1 | cut -d'.' -f1-2)
-REQUIRE_GO_VERSION = 1.20
+REQUIRE_MINIMUM_GO_VERSION = 1.20
```

Comment:
Why do we have this change? Shouldn't all the nodes use the same go versions --  1.21?

Reply:
Don't have the context of this [change](https://github.com/cosmos/gaia/pull/2853/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52). 
I reverted it until it's confirmed that using multiple Go versions is safe.
cc @MSalopek